src/components/BillDisplay.js

```diff
@@ -0,0 +1,33 @@
+const BillDisplay = (props) => {
+    return (
+        <div className="bill-display-container">
+            <div className="gross-bill-per-person">
+                <div className="display-label-container">
+                        <p>Gross Bill</p>
+                        <p className="per-person-label">/ Person</p>
+                </div>
+                <p className="amount-display">{props.grossBillPerPersonState}</p>
+            </div>
+            <div className="tip-per-person">
+                <div className="display-label-container">
+                    <p>Tip Amount</p>
+                    <p className="per-person-label">/ Person</p>
+                </div>
+                <p className="amount-display">{props.tipPerPersonState}</p>
+            </div>
+            <div className="total-bill-per-person">
+                <div className="display-label-container">
+                    <p>Total Bill</p>
+                        <p className="per-person-label">/ Person</p>
```

Comment:
Take a look at the indentation for line 22

Reply:
Done